app/models/attribute/GlobalAttributeTable.scala

```diff
@@ -131,18 +131,28 @@ object GlobalAttributeTable {
     globalAttributes.list
   }
 
+  def toInt(s: Option[String]): Option[Int] = {
+    try {
+      Some(s.getOrElse("6").toInt)
+    } catch {
+      case e: Exception => None
+    }
+  }
+
   /**
-    * Gets global attributes within a bounding box for the public API.
+    * Gets global attributes within a bounding box and a potentially specified severity rating for the public API.
     *
     * @param minLat
     * @param minLng
     * @param maxLat
     * @param maxLng
+    * @param severity
     * @return
     */
-  def getGlobalAttributesInBoundingBox(minLat: Float, minLng: Float, maxLat: Float, maxLng: Float): List[GlobalAttributeForAPI] = db.withSession { implicit session =>
+  def getGlobalAttributesInBoundingBox(minLat: Float, minLng: Float, maxLat: Float, maxLng: Float, severity: Option[String]): List[GlobalAttributeForAPI] = db.withSession { implicit session =>
     val attributes = for {
-      _att <- globalAttributes if _att.lat > minLat && _att.lat < maxLat && _att.lng > minLng && _att.lng < maxLng
+      _att <- globalAttributes if _att.lat > minLat && _att.lat < maxLat && _att.lng > minLng && _att.lng < maxLng &&
+        (_att.severity.isEmpty && severity.getOrElse("") == "none" || severity.isEmpty || _att.severity === toInt(severity))
```

Comment:
For adding a comment for this line, I meant that this particular line is confusing, and a comment that explains exactly what this line is doing inside of the function would be useful.

Reply:
looks good, thank you!